proxy/src/schema/mod.rs

```diff
@@ -73,6 +82,45 @@ impl Mutation {
             metadata: meta.into(),
         })
     }
+
+    fn register_project(
+        ctx: &Context,
+        domain: String,
+        name: String,
+    ) -> Result<registry::Transaction, Error> {
+        // TODO(xla): Get keypair from persistent storage.
+        let fake_pair = ed25519::Pair::from_legacy_string("//Robot", None);
+        // TODO(xla): Remove single-threaded executor once async/await lands in juniper:
+        // https://github.com/graphql-rust/juniper/pull/497
+        let transaction_id = futures::executor::block_on(ctx.registry.register_project(
+            &fake_pair,
+            domain.clone(),
+            name.clone(),
+        ))?;
+
+        Ok(registry::Transaction {
+            id: juniper::ID::new(transaction_id.to_string()),
+            messages: vec![registry::Message::ProjectRegistration(
+                registry::ProjectRegistration {
+                    domain: domain,
+                    name: name,
+                },
+            )],
+            state: registry::TransactionState::Applied(registry::Applied {
+                block: juniper::ID::new(radicle_registry_client::H256::random().to_string()),
```

Comment:
You can get this from [`TransactionApplied::block`](https://github.com/radicle-dev/radicle-registry/blob/8689536f7e30d5e4839141658d2357e2cc550755/client/src/interface.rs#L49)